src/main/resources/application.yaml

```diff
@@ -569,5 +569,6 @@ bulk-action:
 
 SERVICE_AUTH_MICROSERVICE: nfdiv_case_api
 
-toggle:
-  enable_qr_code_reading: ${ENABLE_QR_CODE_READING:true}
+logging:
```

Comment:
Is there a reason you added this?

Reply:
I need to add this when I'm running locally to so that I can see the log messages

Reply:
what impact will it have on production?

Reply:
I don't how how logging it configred in production, I have removed this for now.